IdentityCore/src/broker_operation/request/MSIDBrokerOperationRequest.m

```diff
@@ -106,9 +123,23 @@ - (NSDictionary *)jsonDictionary
     return json;
 }
 
--(NSString *)logInfo
+- (NSString *)logInfo
 {
     return [NSString stringWithFormat:@"%@",[self.jsonDictionary msidMaskedRequestDictionary]];
 }
 
+- (BOOL)shouldIgnoreBrokerKey
+{
+#if !TARGET_OS_OSX
+    return NO;
+#else
```

Comment:
nit: we can remove this "#else"

Reply:
Removed

Reply:
Actually if I removed the else, there is an API only available for Mac and will break the build on iOS